.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,70 @@
+<!--  All Submissions -->
+
+
+## Community Note
+<!-- Please leave the community note as is. -->
+* Please vote on this PR by adding a :thumbsup: [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original PR to help the community and maintainers prioritize for review
+* Please do not leave "+1" or "me too" comments, they generate extra noise for PR followers and do not help prioritize for review
+
+
+## PR Checklist
+
+- [ ] Have you followed the guidelines in our [Contributing Documentation](../contributing/README.md)?
```

Comment:
should this be I have instead of Have you?
```suggestion
- [ ] I have followed the guidelines in our [Contributing Documentation](../contributing/README.md)
```

Reply:
I think they all need to be either first person or second person, not some of each. I will change them all to first person and on re-review LMK if it sounds better.